tito.spec

```diff
@@ -23,8 +23,8 @@ Summary: A tool for managing rpm based git projects
 
 Group: Development/Tools
 License: GPLv2
-URL: http://rm-rf.ca/tito
-Source0: http://rm-rf.ca/files/tito/tito-%{version}.tar.gz
+URL: https://github.com/dgoodwin/tito
+Source0: https://github.com/dgoodwin/tito/archive/tito-%{version}-1.tar.gz
```

Comment:
But this is a binary different source than what `tito build --tgz` produce.

Reply:
Sure, but it has the significant advantage of *actually existing*. The other URL does not (any more). We work with what we can. :)

I did do a quick eyeball compare of the two and they look extremely similar, and the github-generated tarball works fine to actually do a package build. I didn't do an actual diff on the contents, though.